de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/SarosSessionManager.java

```diff
@@ -333,7 +333,7 @@ public void stopSession(SessionEndReason reason) {
             sessionEnding(session);
 
             try {
-                session.stop();
+                session.stop(reason);
```

Comment:
Another option would be to do the check which reason caused the session end here and then just pass a boolean `notifyParticipants` to `SarosSession#stop()`.

I'm split on which option would be better.